buildSrc/src/main/kotlin/kotlin-library-convention.gradle.kts

```diff
@@ -58,6 +58,15 @@ kotlin {
         else -> throw GradleException("Host OS is not supported in Kotlin/Native.")
     }
 
+    listOf(
+        iosX64(),
+        iosArm64(),
+    ).forEach {
+        it.binaries.framework {
+            baseName = "mutator"
```

Comment:
Now I'm unsure about this line. Does that mean the binaries for each module will have the same base name? 🤔 Will that be an issue? I can publish local & run still, so maybe not an issue.

Reply:
lol I'm not sure but let's give it a whirl!